docs/migrate-to-jina3/migrate.md

```diff
@@ -0,0 +1,72 @@
+# Migration to Jina 3
+
+Jina 3 introduces a number of exciting features, but to be able to enjoy them, you will also have to make some
+tweaks to your existing Jina 2 code.
+
+One of the major changes in Jina 3 is the inclusion of the [docarray library](https://docarray.jina.ai/).
+Accordingly, the scope of the changes in Jina 3 will be mainly related to `Document` and `DocumentArray`.
+Here you can find out what exactly you will have to change.
+
+## Attribute renamings
```

Comment:
Could we rename some of these titles to highlight the benefit of the change?

e.g. `More natural attribute names`

Not sure we can do it for every title, but let's give it a shot